seed/seed.go

```diff
@@ -181,6 +181,14 @@ type SnapHandler interface {
 	HandleUnassertedSnap(name, path string, tm timings.Measurer) (newPath string, err error)
 }
 
+// A AutoImportAssertionsLoaderSeed can be used to import all auto import assertions
+// via LoadAutoImportAssertions.
+type AutoImportAssertionsLoaderSeed interface {
+	// LoadAutoImportAssertions attempts to loads all Auto import assertions
+	// from the root of the seed. This is the best effort and function does not fail.
```

Comment:
I think s/the best effort/a best effort/ is more grammatical here

Reply:
Changed